monitoring/ceph-mixin/dashboards/cephfs.libsonnet

```
@@ -18,7 +18,7 @@ local u = import 'utils.libsonnet';
                            1,
                            '$datasource')
         .addTargets(
-          [u.addTargetSchema(expr, 1, 'time_series', legendFormat)]
+          [u.addTargetSchema(expr, '', 1, 'time_series', legendFormat)]
```

Following my suggestion of using default args and reordering the method signature, this would leave like:
```suggestion
          [u.addTargetSchema(expr, legendFormat)]
```
Let's try to simplify and reduce as much as possible the boilerplate code.

---

Done.